web-app/src/components/ratings-dropdown-review.tsx

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import styled from "styled-components";
+import StarRatings from "react-star-ratings";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { translations } from "src/translations";
+import { faCaretDown } from "@fortawesome/free-solid-svg-icons";
+import { StarRatings as StartRatingsType } from "generated/graphql-operations";
+import * as urlGenerators from "src/pages/url-generators";
+import { Link } from "react-router-dom";
+
+const T = translations.legacyTranslationsNeedsRefactor;
+
+const StarRatingsDropdownButton = styled.a`
+	position: relative;
+	display: inline-block;
+	vertical-align: middle;
+	padding-top: 4px;
+	padding-bottom: 6px;
+`;
+
+const NumRatingsText = styled(Link)`
```

Comment:
Change name to "AverageRatingValue" and this doesn't need to be a link button, it can be a span